scripts/perf_build.sh

```diff
@@ -6,7 +6,7 @@
 set -euo pipefail
 
 # downloading the zip because the git is very large (this is also very large, but still smaller)
-curl -SL https://codeload.github.com/Granulate/linux/zip/1ebd66ff4664ad88421e4b1c7294e4f8987b36e1 -o linux.zip
+curl -SL https://codeload.github.com/marcin-ol/linux/zip/776f79af7a8f0bbe27548d8680208196c6f4be9a -o linux.zip
```

Comment:
Post the [merge](https://github.com/Granulate/linux/pull/2) you can upgrade it now :)